updating GCP workload identity docs (#1107)
astronomer/docs: astro/connect-external-services.md

    ```text
    astro-nuclear-science-2730@astronomer-prod.iam.gserviceaccount.com
    ```

> 3. Navigate to the Google Cloud project in which your external data service is hosted.

Reviewer: ```suggestion
```

@melugoyal Could we just get rid of step 3 if the user is going straight to running the gcloud CLI command? What do they need from the GCP console at this point in the process? The name of the project maybe?

Author: yeah i was also thinking step 3 felt extraneous. good call

> 4. Grant the Google service account for your Astro Deployment a role that has access to your external data service.

Reviewer: ```suggestion
4. Grant the Google service account for your Astro Deployment an IAM role that has access to your external data service. With the Google Cloud CLI, run:
```

    ```text

>     gcloud projects add-iam-policy-binding $GOOGLE_CLOUD_PROJECT --member=serviceAccount:astro-nuclear-science-2730@astronomer-prod.iam.gserviceaccount.com --role=roles/viewer

Reviewer: ```suggestion
    gcloud projects add-iam-policy-binding $GOOGLE_CLOUD_PROJECT --member=serviceAccount:astro-<deployment-namespace>@<gcp-project-name>.iam.gserviceaccount.com --role=roles/viewer
```

    ```

>     Instructions to do this from the GCP console can be found [here](https://cloud.google.com/iam/docs/grant-role-console#grant_an_iam_role).

Reviewer: ```suggestion
    For instructions on how to grant your service account an IAM role in the Google Cloud console, see [Grant an IAM role](https://cloud.google.com/iam/docs/grant-role-console#grant_an_iam_role).
```

5. Optional. Repeat these steps for every Astro Deployment that requires access to external data services on GCP.

:::info

GCP has a 30-character limit for service account names. For Deployment namespaces which are longer than 24 characters, use only the first 24 characters when determining your service account name.

For example, if your Google Cloud project is named `astronomer-prod` and your Deployment namespace is `nuclear-scintillation-2730`, the service account name is:

```text
astro-nuclear-scintillation-27@astronomer-prod.iam.gserviceaccount.com
```

> This can be generated by running:

Reviewer: ```suggestion
To determine your Deployment namespace that meets requirements without having to manually count characters, run:

```
